src/app/pages/home/about/about.component.html

```diff
@@ -1 +1,41 @@
-<div>About Page</div>
\ No newline at end of file
+<div class="aboutus-overlay1">
+    <div class="aboutus-overlay1-content">
+      <div class="headline">About NTU Open Source Society</div>
+      <p>NTU Open Source Society (OSS) is a student community at the Nanyang Technological University helping to promote the value and importance of Open Source in NTU. We attract students of common technical interests to foster a community of engineers, hackers and builders to explore their interests in technology and share their knowledge and expertise with others. Through our weekly meetings, we help students learn practical coding skills, find partners to work with on interesting projects and also help promote the value and importance of contributing to open source development. Regardless of skill level or expertise, if you like to build things, this is the place for you.</p>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="column">
+                <div class="container-content">
+                    <img src="../../../../assets/pages/about/about-us_container_code.png" width="48" height="48" alt="">
```

Comment:
I would suggest to remove the `width` and `height` attributes here as we can also define it from the stylings. Consider giving it some bottom padding as well.

```scss
img {
    height: 5rem;
    padding-bottom: 1rem;
}
```